gen_wrap.py

```diff
@@ -1417,6 +1417,7 @@ def write_wrappers(expf, wrapf, methods):
 
 def gen_wrapper(include_dirs, include_barvinok=False, isl_version=None):
     fdata = FunctionData(["."] + include_dirs)
+    fdata.read_header("isl/ctx.h")
```

Comment:
Could you also commit the preprocessed header cache? This is so that we don't need to have `pcpp` as an install-time dependency.

Reply:
Done.